examples/chef/common/chef-rvc-operational-state-delegate.cpp

```diff
@@ -62,10 +62,26 @@ CHIP_ERROR RvcOperationalStateDelegate::GetOperationalPhaseAtIndex(size_t index,
 
 void RvcOperationalStateDelegate::HandlePauseStateCallback(GenericOperationalError & err)
 {
```

Comment:
Please use the `GetInstance()->UpdateCountdownTimeFromDelegate();` calls as seen in examples/all-clusters-app/all-clusters-common/src/operational-state-delegate-impl.cpp and match when they are called, otherwise there will not be proper management of countdown time.

Reply:
added, thank you